test/arcade/potts/sim/PottsSeriesTest.java

```diff
@@ -639,6 +645,9 @@ public void updatePopulation_multiplePopulations_createsMap() {
         assertEquals(1, series.populations.get(POPULATION_ID_1).getInt("CODE"));
         assertEquals(2, series.populations.get(POPULATION_ID_2).getInt("CODE"));
         assertEquals(3, series.populations.get(POPULATION_ID_3).getInt("CODE"));
+        assertEquals(DEFAULT_CELL_CLASS, series.populations.get(POPULATION_ID_1).get("CLASS"));
+        assertEquals(DEFAULT_CELL_CLASS, series.populations.get(POPULATION_ID_2).get("CLASS"));
+        assertEquals(DEFAULT_CELL_CLASS, series.populations.get(POPULATION_ID_3).get("CLASS"));
```

Comment:
The above method `updatePopulation_onePopulation_createsMap` also needs a check for `DEFAULT_CELL_CLASS`